tests/base_test_case.py

```diff
@@ -1,13 +1,21 @@
 import os
 from unittest import TestCase
 from unittest.mock import patch
-
+from pytest import fixture
+from pytest_mock import MockerFixture
 from electionguard.constants import PrimeOption
 
 
 class BaseTestCase(TestCase):
     """Base Test Case for overriding environment variables."""
 
+    mocker: MockerFixture
```

Comment:
Expose a mocker object to all tests since it's cumbersome to do and requires a pylint disable